crypto/test/bn_test_lib.c

```diff
@@ -133,7 +133,7 @@ int BN_rand(BIGNUM *rnd, int bits, RAND *rng) {
   bit = (bits - 1) % 8;
   mask = 0xff << (bit + 1);
 
-  buf = OPENSSL_malloc(bytes);
+  buf = (uint8_t*) OPENSSL_malloc(bytes);
```

Comment:
I've tried to compile this as c++ code. In this case g++ complains that `void*` is no `uint8_t*`.
```error: invalid conversion from ‘void*’ to ‘uint8_t*‘[-fpermissive]```
It's not needed now, so I've removed this from the current version.